forwarding/src/test/kotlin/com/pkware/file/forwarding/StubPath.kt

```diff
@@ -0,0 +1,9 @@
+package com.pkware.file.forwarding
```

Comment:
i would like to make the forwarding classes internal but i tried that and I get a compiler error for exposing internal members of Forwarding in a public class

the stub path and filesystem are used for only tests right now and I am okay with publically exposing forwarding filesystem since it only forwards calls